src/components/LayerList/LayerForm.tsx

```diff
@@ -16,14 +15,13 @@ function LayerForm({ config, updateLayer }: LayerFormProps) {
     return <p>Loading...</p>
   }
 
-  const cubeVariables = collection && collection.stac['cube:variables'];
-  const datetime_range = collection && collection.datetime_range;
-
-  if (cubeVariables) {
-    return <LayerFormWithDateSlider config={config} collection={collection.stac} updateLayer={updateLayer} />
+  if (!collection) {
+    return <p>Collection not found.</p>;
   }
 
-  if (datetime_range) {
+  const timeseries_type = collection.timeseries_type;
+
+  if (timeseries_type === 'forecast') {
     return <LayerFormWithDatePicker config={config} collection={collection.stac} updateLayer={updateLayer} />
   }
```

Comment:
TODO: add timeseries_type === 'historical'. What type of date time selection should be possible? All hours for all past days may not work for a small slider.

Reply:
Maybe we could have a calendar input and an hour slider with marks for hours 0-23 and then a set of forward/backward arrows that would let the user click from hour to hour. If the arrows get clicked the selected values in the calendar/slider would get updated.

Reply:
That's a good idea, I'm not sure if the arrows are necessary as you can go forward and backward on the slider using the arrow keys on the keyboard, but that may not be obvious to folks. I am imaging that the UI for datetimes for historical data looks exactly like it does for forecast data, except when any given day is selected the slider has hours 0-23 which would represent the analysis / 0 hour forecasts at each hour of the day (not the forecasted values from midnight).